soc/arm/silabs_exx32/Kconfig

```diff
@@ -134,6 +134,27 @@ choice SOC_GECKO_EMU_DCDC_MODE
 		bool "Bypass"
 endchoice
 
+config SOC_GECKO_DEV_INIT
+	bool
+	default n
```

Comment:
The value n is the default in Kconfig, so this line can be removed (also line 146 and line 153 in this file)